### PR TITLE
Razor pay

### PR DESCRIPTION
**Pull request by oveshparasara**
- Razor pay intigration completed

### CODE REVIEW

**Review by anshul-031 — APPROVED** [review]

File: src/app/payments/razor-pay/razor-pay.component.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      }
+    );
+  }
+
+  @HostListener('window:payment.success', ['$event'])
+  onPaymentSuccess(event: any): void {
+       console.info("payment successs")
+  }
+
+  logout(): void {
```
Comment (gurpreetlogiciel) [inline src/app/payments/razor-pay/razor-pay.component.ts:123]: @oveshparasara can we move this method in user service and direct user from there ?

File: src/app/payments/razor-pay/razor-pay.component.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+export class RazorPayComponent implements OnInit {
+
+  constructor(public userService: UserService, private paymentService: PaymentService, private userAuthService: UserAuthService, private router: Router,) { }
+
+  ngOnInit(): void {
+  }
+
+
+  payNow() {
+    const orderData =  {
```
Comment (gurpreetlogiciel) [inline src/app/payments/razor-pay/razor-pay.component.ts:23]: @oveshparasara is this a temporary object? seems the data is static now

**Review by gurpreetlogiciel — COMMENTED** [review]
@oveshparasara can you check my comments once. all are ok just confirm me

File: src/app/payments/razor-pay/razor-pay.component.ts (hunk 2 above)
Comment (oveshparasara) [reply src/app/payments/razor-pay/razor-pay.component.ts:23]: yes just now static data

File: src/app/payments/razor-pay/razor-pay.component.ts (hunk 1 above)
Comment (oveshparasara) [reply src/app/payments/razor-pay/razor-pay.component.ts:123]: will move in next commit

Comment (gurpreet-developer) [reply src/app/payments/razor-pay/razor-pay.component.ts:123]: ok @oveshparasara

File: src/app/payments/razor-pay/razor-pay.component.ts (hunk 2 above)
Comment (gurpreet-developer) [reply src/app/payments/razor-pay/razor-pay.component.ts:23]: ok @oveshparasara

**Pull request merged**